role-based-access-system/src/routes/permission.route.js

```diff
@@ -0,0 +1,82 @@
+import { Router } from "express";
+const route = Router();
+import query from "../db/query.js";
+
+// get all permissions for a role.
+route.get("/roles/permissions/:roleId", async (req, res) => {
```

Comment:
Why role route is in permissions routes file ?

Reply:
This is not fixed yet roles api should not be in permissions routes

Reply:
Changed role api to role route